charts/dapr/charts/dapr_placement/templates/dapr_placement_deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
           initialDelaySeconds: {{ .Values.debug.initialDelaySeconds }}
 {{- end }}
           periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
-          failureThreshold: 5
+          failureThreshold: 30
```

Comment:
What is the root cause of failure? If related to workloads, we should add it as a configurable value.